config.tf

```
@@ -1,6 +1,12 @@
+locals {
+  demo_or_es   = "${var.postgresql_address != "" ? "external_services" : "demo"}"
```

same comment as https://github.com/hashicorp/terraform-aws-terraform-enterprise/pull/18/files#r337215666

but for something like 
```suggestion
  app_data_mode   = "${var.postgresql_address != "" ? "external_services" : "demo"}"
  
  or something to say what it is better.
```